### DISCUSSION TITLE
Slow plotting even on NVME

### DISCUSSION THREAD

**Discussion by T4C-FanBoy** [Chia-Network/chia-blockchain#4635] [Support]
[
![SlowPlotting](https://user-images.githubusercontent.com/39564985/117915401-356a9680-b2b3-11eb-9f44-4c4685c9b512.png)
](url)

The image explains everything. I purchased a 1TB NVME and when using CrystalDiskMark , I get 4000MB/s and it never goes down.

When using Chia, the drives is limited to 400MB/s in Phase 1 and to 80-180MB/s in table 2.

Is this normal??

**Comment by ilyakooo0** [comment]
I experience the same behavior. I assume that the drive speed is not the bottleneck in this situation.

**Comment by radvansky-tomas** [comment]
You need to be able to generate meaningful 4000MB /s to actually get into NVME bottleneck

**Comment by ghost** [comment]
I plot on my M1 Mac Mini, 16GB Ram, using external Samsung T7 SSDs.
I get F1 complete times between 85 and 150 seconds. Sorting seem to happen always with 0.281.


```
Plot size is: 32
Buffer size is: 3389MiB
Using 128 buckets
Using 4 threads of stripe size 65536

Starting phase 1/4: Forward Propagation into tmp files... Mon May 10 11:05:50 2021
Computing table 1
F1 complete, time: 84.916 seconds. CPU (129.77%) Mon May 10 11:07:15 2021
Computing table 2
        Bucket 0 uniform sort. Ram: 3.243GiB, u_sort min: 1.125GiB, qs min: 0.281GiB.
        Bucket 1 uniform sort. Ram: 3.243GiB, u_sort min: 0.563GiB, qs min: 0.281GiB.
        Bucket 2 uniform sort. Ram: 3.243GiB, u_sort min: 0.563GiB, qs min: 0.281GiB.
        Bucket 3 uniform sort. Ram: 3.243GiB, u_sort min: 0.563GiB, qs min: 0.281GiB.
        Bucket 4 uniform sort. Ram: 3.243GiB, u_sort min: 1.125GiB, qs min: 0.281GiB.
        Bucket 5 uniform sort. Ram: 3.243GiB, u_sort min: 0.563GiB, qs min: 0.281GiB.
        Bucket 6 uniform sort. Ram: 3.243GiB, u_sort min: 0.563GiB, qs min: 0.281GiB.
        Bucket 7 uniform sort. Ram: 3.243GiB, u_sort min: 1.125GiB, qs min: 0.281GiB.
```

**Comment by T4C-FanBoy** [comment]
What can be the bottleneck? It uses 30% of the cpu , 10% of the memory and 8% of the nvme.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
